src/main/java/io/supertokens/utils/Utils.java

```diff
@@ -279,8 +279,8 @@ public static class PubPriKey {
         public PubPriKey(String s) {
             // We split by both | and ; because in old versions we used to use ";" in dynamic and "|" in static keys
             // Now we are consolidating all of them to use "|", but by handling legacy keys, we can avoid the need for manual key migration.
-            // I.e.: this way only people who used access_token_signing_key_dynamic has to do it instead of everyone.
-            // for everyone else, the key rotation should get it done in a few weeks.
+            // I.e.: this way only people who used access_token_signing_key_dynamic has to do manual migration instead of everyone.
+            // for everyone else, the key rotation should get it done.
```

Comment:
most people will have access_token_signing_key_dynamic to true - since that's the default. Is there anyway to not make them to manual migration?